src/object/object_primitive.h

```diff
@@ -348,5 +348,11 @@ extern int pr_area_init (void);
 extern void pr_area_final (void);
 
 extern int pr_complete_enum_value (DB_VALUE * value, TP_DOMAIN * domain);
+extern int or_get_compression_length (char *string, int charlen);
+extern int do_get_compressed_data_from_buffer (OR_BUF * buf, char *data, int compressed_size, int decompressed_size);
+
+#define MINIMUM_LENGTH_FOR_COMPRESSION 255
+#define TEMPORARY_DISK_SIZE 256
+#define SIZE_OF_COMPRESSION_LENGTHS 4
```

Comment:
@lucianlazar2211 

I'd like to have more explicit name with module prefix, for instance, `PRIM_MINIMUM_STRING_LENGTH_FOR_COMPRESSION`.